charts/zabbix/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-apiVersion: v2 # Don't change this
+apiVersion: v2  # Don't change this
 name: zabbix
-version: 6.1.2 # helm chart version
+version: 6.1.2  # helm chart version
 # LTS Zabbix version by default due to stability. See: https://www.zabbix.com/life_cycle_and_release_policy
-appVersion: 7.0.6 # zabbix version
+appVersion: 7.0.6  # zabbix version
```

Comment:
Hi @fibbs!

I suggest changing to 7.0.8. This change can be made in a new PR before publishing a new release.

Reply:
Yes, should be raised to 7.0.8. Didn't want to do so within this PR, though. This specific diff is not because of version change (which would happen outside PR) but because of the space before comment. Did linting with `ct` tool, which demands more than one space before in-line comment

Reply:
Ok, @fibbs! Great.